buildSrc/src/main/groovy/generate-pom.gradle

```diff
@@ -23,6 +23,7 @@
  * (INCLUDING NEGLIGENCE OR OTHERWISE) ARISING IN ANY WAY OUT OF THE USE
  * OF THIS SOFTWARE, EVEN IF ADVISED OF THE POSSIBILITY OF SUCH DAMAGE.
  */
+//file:noinspection GroovyVariableCanBeFinal
```

Comment:
So we have both this inspection suppression and finals below. Can we please stick to one approach? E.g. disable this check for finals in IDEA config completely?

Reply:
This is a temporary measure to reduce the noise in Groovy files, and in this one in particular. 
In other places, the warning helps to see that variables are not modified in (such a shaky Groovy) code.
Turning off this warning altogether is a bit too much.
As we're going to migrate these scripts to Kotlin, we'd be addressing these things.